src/Rubric/RubAbstractTextArea.class.st

```diff
@@ -109,7 +109,7 @@ Class {
 
 { #category : #settings }
 RubAbstractTextArea class >> backgroundColor [
-	^ BackgroundColor ifNil: [ BackgroundColor := Color white] 
+	^ BackgroundColor ifNil: [ BackgroundColor := self theme backgroundColor ] 
```

Comment:
Hi,

I don’t know if it’s done somewhere, but when the theme change, this should also be updated I think? Because if the defensive strategy is executed in the light theme, then we change to the dark theme then we need the background to not be white :)